src/python/deepgnn/pytorch/common/args.py

```diff
@@ -59,3 +61,68 @@ def init_common_args(parser: argparse.ArgumentParser):
     init_dataset_args(parser)
     init_optimizer_args(parser)
     init_other_args(parser)
+
+
+def init_trainer_args(parser: argparse.ArgumentParser):
```

Comment:
why do you need to move arguments from training module to common module?

Reply:
also, why it is not in a change log?

Reply:
So we can delete pytorch/training

Reply:
Please update change log.